[DOCS] Updates anomaly detection alert docs with the new alerting terminology (#70486)
elastic/elasticsearch: docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

beta::[]

> {anomaly-detect-cap} alerting runs scheduled checks on an {anomaly-job} or a 

Reviewer: Since [https://www.elastic.co/guide/en/kibana/master/alerting-getting-started.html#alerting-concepts-differences](Kibana alerting docs) already describe "Kibana alerting" vs "Elasticsearch alerting", I think we should try to avoid the idea that there's another type of "ML alerting" or "anomaly detection alerting".  If you agree, I suggest something like this: 
```suggestion
{kib} {alert-features} include support for {ml} rules, which runs scheduled checks on an {anomaly-job} or a 
```

group of jobs to detect anomalies with certain conditions. If an anomaly meets 

> the rules you defined, an alert is created and the associated action is 

Reviewer 2: Not sure if we should use the word `rules` here, might be confusing with the new terminology, because `alert` has been renamed to `rule`.

example, you can create a rule to check an {anomaly-job} every fifteen minutes 

> for critical anomalies and to notify you in an email. This page helps you to 

Reviewer: I think this statement is no longer accurate (see comment about title) and is redundant anyway:
```suggestion
for critical anomalies and to notify you in an email. 
```

> configure {anomaly-detect} alerting. To learn more about alerting in the 

Reviewer: I think it's worth qualifying that we're sending them to the Kibana-related docs:
```suggestion
To learn more about {kib} {alert-features}, 
```

> configure {anomaly-detect} alerting. To learn more about alerting in the 

Author: ```suggestion
To learn more about alerting in the 
```

> {stack}, refer to 

Reviewer: Continuation of edit on previous half of sentence:
```suggestion
refer to 
```

> {kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting and Actions].

Reviewer 2: I don't know if we're in control of all links, but in the preview, I see `general alert details.`, but it should be `general rule details.`

[[creating-anomaly-alert-rules]]

> == Creating rules

Reviewer 2: maybe singular would be better here 
```suggestion
== Creating a rule
```

> You can create {anomaly-detect} alerting rules in the {anomaly-job} wizard after 

Reviewer: Not mandatory, but since they're just called "machine learning rules" in the Kibana docs, maybe that's sufficient here too:
```suggestion
You can create {ml} rules in the {anomaly-job} wizard after 
```

you start the job, from the job list, or under **{stack-manage-app} > 
{alerts-ui}**. On the *Create rule* window, select *{anomaly-detect-cap} alert* 

> under the {ml-cap} section, then give a name to the rule and optionally provide 

Reviewer: It looks odd to only have the M capitalized here, so I'd suggest either using "Machine Learning" or "machine learning"
```suggestion
under the {ml} section, then give a name to the rule and optionally provide 
```

tags.

Specify the time interval for the rule to check detected anomalies. It is 
recommended to select an interval that is close to the bucket span of the 
associated job. You can also select a notification option by using the _Notify_ 
selector. An alert remains active as long as anomalies are found for a 
particular {anomaly-job} during the check interval. When there is no anomaly 
found in the next interval, the `Recovered` action group is invoked and the 
status of the alert changes to `OK`. For more details, refer to the 
documentation of 
{kibana-ref}/defining-alerts.html#defining-alerts-general-details[general rule details].
  
[role="screenshot"]
image::images/ml-anomaly-alert-type.jpg["Creating a rule for an anomaly detection alert"]
  
Select the {anomaly-job} or the group of {anomaly-jobs} that is checked against 
the rule. If you assign additional jobs to the group, the new jobs are 

> automatically checked the next time when the conditions are checked.

Reviewer: ```suggestion
automatically checked the next time the conditions are checked.
```

The `anomaly_score` indicates the significance of a given anomaly compared to 
previous anomalies. The default severity threshold is 75 which means every 
anomaly with an `anomaly_score` of 75 or higher triggers the associated action.

You can select whether you want to include interim results. Interim results are 
created by the {anomaly-job} before a bucket is finalized. These results might 
disappear after the bucket is fully processed. Include interim results if you 
want to be notified earlier about a potential anomaly even if it might be a 
false positive. If you want to get notified only about anomalies of fully 
processed buckets, do not include interim results.

You can also test the configured conditions against your existing data and check 
the sample results by providing a valid interval for your data. The generated 
preview contains the number of potentially created alerts during the relative 
time range you defined.

[[defining-actions]]
== Defining actions

As a next step, connect your rule to actions that use supported built-in 
integrations by selecting a connector type. Connectors are {kib} services or 

> third-party integrations that perform an action when the alert conditions are 

Reviewer: I think per [Connectors and actions](https://www.elastic.co/guide/en/kibana/master/action-types.html) it's rule conditions now not alert conditions:
```suggestion
third-party integrations that perform an action when the rule conditions are 
```
